lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
```

Comment:
So move the `schemas.js` from `lib/schemas.js` to `lib/collections/schemas.js` ?